xlator/global.go

```diff
@@ -0,0 +1,15 @@
+package xlator
+
+// AllOptions contains all possible xlator options for all xlators
+// Other packages can directly import this
+var AllOptions map[string][]Option
+
+// InitOptions initializes the global variable xlator.AllOptions
+func InitOptions() error {
```

Comment:
This is great.

Along with an options table, I would like an xlator table/map as well. This will allow volgen to easily map options and set defaults when required. 

Something like,
```go
type Xlator struct {
  Id string
  Options []Option
}

var Xlators map[string]Xlator
```

This can even be the single table removing the need for the `AllOptions` table. Can you do this?

Or if you prefer, I can merge this patch, and do the required changes myself with the volgen change.